index.js

```diff
@@ -128,5 +128,29 @@ ansiEscapes.image = (buffer, options = {}) => {
 };
 
 ansiEscapes.iTerm = {
-	setCwd: (cwd = process.cwd()) => `${OSC}50;CurrentDir=${cwd}${BEL}`
+	setCwd: (cwd = process.cwd()) => `${OSC}50;CurrentDir=${cwd}${BEL}`,
+	/**
+	* @description iTerm-specific annotations
+	* @param {string} message - Annotation message
+	* @param {Object} [options] - Options
+	* @param {number} [options.length] - Nonzero number of cells to annotate. Defaults to remainder of line
+	* @param {number} [options.x] - Starting X coordinate; defaults to cursor position; must be used with `y` and `length`
+	* @param {number} [options.y] - Starting Y coordinate; defaults to cursor position; must be used with `x` and `length`
+	* @param {boolean} [options.hidden] - If true, this is a "hidden" annotation
+	* @returns {string} Escape code for annotation
+	*/
```

Comment:
Add it to https://github.com/sindresorhus/ansi-escapes/blob/master/index.d.ts instead of here.